source/js/options.js

```diff
@@ -129,20 +129,27 @@ $(document).ready(function() {
   showReviewsLink();
   load_options();
 
-  $.ajax("ChangeLog").done(function(text) {
-    $("#changelogText").text(text);
-  });
+  var changeLogRequest = new XMLHttpRequest();
+  changeLogRequest.onreadystatechange = function() {
+    if (changeLogRequest.responseText) {
+      $("#changelogText").text(changeLogRequest.responseText);
+    }
+  }
+  changeLogRequest.open('GET', 'ChangeLog', true);
+  changeLogRequest.send();
```

Comment:
I haven't found any details, but `$.ajax` fails. On stackoverflow I have found only something about CSP for injected scripts, which is not the case here. I expect it does not use only `XMLHttpRequest` on the background and gets blocked. Using `fetch()` sounds reasonable.